README.md

```diff
@@ -1,7 +1,7 @@
 # Safe Apps SDK
 
 [![Logo](https://raw.githubusercontent.com/gnosis/safe-apps-sdk/master/assets/logo.png)](https://gnosis.pm/)
```

Comment:
Should we add a newline? ... the preview shows some weird alignment with the logo